File: 01 - Multiples of 3 or 5/rust/main.rs
fn main()
{
    let result: i32 = multiples_of_3_and_5(1000);
    println!("result: {}", result);
}

fn multiples_of_3_and_5(limit: i32) -> i32
{
    let mut sum: i32 = 0;
    for i in 1..limit
    {
        if i % 3 == 0 || i % 5 == 0
        {
            println!("{}", i);
            sum += i;
        }
    }
    return sum;
}

File: 01 - Multiples of 3 or 5/nim/main.nim
proc multiples_of_3_and_5(limit: int): int =
    var sum: int = 0
    for i in 1 ..< limit:
        if i mod 3 == 0 or i mod 5 == 0:
            echo i
            sum += i
    sum

let result: int = multiples_of_3_and_5(1000)
echo "result: ", result